inc/package.class.php

```diff
@@ -190,7 +194,7 @@ public function prepareInputForAdd($input) {
       }
       try {
          $uploadedFile = $preparedFile['uploadedFile'];
-         $input['filename'] = 'flyvemdm/package/' . $this->fields['entities_id'] . '/' . uniqid() . '_' . basename($uploadedFile);
+         $input['filename'] = 'flyvemdm/package/' . $input['entities_id'] . '/' . uniqid() . '_' . basename($uploadedFile);
```

Comment:
not related to fdroid: when creating a package, $this->fields should be empty. There is an error here.